src/XmlUtility.ps1

```diff
@@ -0,0 +1,128 @@
+﻿$WikiConfig = @{
+    CacheDir = Resolve-FullPath $Config.ArtifactsDir 'cache'
+    WikiDir = Resolve-FullPath $Config.RootDir '..\..\SpbDotNet.wiki'
+}
+
+$WikiRepository = @{
+    Communities = @{}
+    Meetups = @{}
+    Talks = @{}
+    Speakers = @{}
+    Friends = @{}
+    Venues = @{}
+}
+
+function Test-WikiEnvironment()
```

Comment:
You don't use artifacts that created this method. Do you really need it?

Reply:
No, I've removed it